src/main/java/edu/cornell/kfs/fp/batch/xml/AccountingXmlDocumentAccountingLine.java

```diff
@@ -0,0 +1,139 @@
+package edu.cornell.kfs.fp.batch.xml;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.XmlType;
+import javax.xml.bind.annotation.adapters.NormalizedStringAdapter;
+import javax.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
+
+import org.apache.commons.lang.StringUtils;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+
+import edu.cornell.kfs.sys.xmladapters.KualiDecimalXmlAdapter;
+
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlType(name = StringUtils.EMPTY, propOrder = {

```

Comment:
I am not sure if we want to enforce an order here, or on the other POJOs

Reply:
Ok, I'll remove these ordering restrictions.